package.json

```diff
@@ -78,6 +78,7 @@
     "makeup-navigation-emitter": "~0.3.8",
     "makeup-prevent-scroll-keys": "~0.0.4",
     "makeup-roving-tabindex": "~0.3.7",
+    "makeup-switch-class": "~0.0.1",
```

Comment:
Just a thought, it would be good to have a group project which imports all the makeup classes, at least for this and ebayui. Having to add a new 5-10 packages and maintaining them would be kinda difficult.

Reply:
Yeah, a seed package might be a good idea, if that's what you mean, although I thought their use was generally frowned upon. I forget where I read that...